cmake/addons/CMakeLists.txt

```diff
@@ -254,11 +254,16 @@ foreach(addon ${addons})
     # try to find a perfect match
     list(FIND ADDONS_TO_BUILD ${id} idx)
     if(idx GREATER -1 OR "${ADDONS_TO_BUILD}" STREQUAL "all")
+      message(STATUS "Exact Match ${id}, building addon")
       set(ADDON_FOUND TRUE)
     # Maybe we have a regex
-    elseif(id MATCHES "${ADDONS_TO_BUILD}")
-      message(STATUS "Pattern ${ADDONS_TO_BUILD} matches ${id}, building addon")
-      set(ADDON_FOUND TRUE)
+    else()
+      foreach(ADDONLISTITEM ${ADDONS_TO_BUILD})
+        if(id MATCHES "${ADDONLISTITEM}")
+          message(STATUS "Pattern ${ADDONLISTITEM} matches ${id}, building addon")
+          set(ADDON_FOUND TRUE)
```

Comment:
Think whould be good to add a `break()` below, as after a `TRUE` the next need no more checked.

Reply:
Will add this, cheers